crates/orderbook/src/main.rs

```diff
@@ -38,7 +38,7 @@ use shared::{
     },
     baseline_solver::BaseTokens,
     current_block::current_block_stream,
-    http_solver::{DefaultHttpSolverApi, SolverConfig},
+    http_solver::{DefaultHttpSolverApi, OptimizeFor, SolverConfig},
```

Comment:
I would probably call this `Objective` instead of `OptimizeFor` in order to make the connection to the respective solver parameter more obvious.

Reply:
👍 - renamed